api/src/Storage.ts

```diff
@@ -48,11 +48,11 @@ export class GearProgramStorage {
   async getProgramPages(programId: HexString, program: GearCommonActiveProgram, at?: HexString): Promise<IGearPages> {
     const pages = {};
     const query =
-      this._api.specVersion >= 1020
+      this._api.specVersion >= 1030
```

Comment:
nit: `1030` can be moved to configs or const

Reply:
Yes, thanks